app/models/custom_org_link.rb

```diff
@@ -6,6 +6,14 @@ class CustomOrgLink < ApplicationRecord
   validates :text, length: {maximum: TEXT_MAX_LENGTH}
   validates :active, inclusion: {in: [true, false]}
   validates :url, url: true
+
+  before_save :trim_name
+
+  private
+
+  def trim_name
+    self.text = text.strip if text.present?
```

Comment:
I would like a spec to go with this change, as with all changes. Can you add a spec in a follow up PR? Thanks in advance! :)